name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/ConsorsbankPDFExtractor.java

```diff
@@ -293,6 +293,13 @@ private void addDividendeTransaction()
                     t.setShares(asShares(v.get("shares")));
                     t.setSecurity(getOrCreateSecurity(v));
                 })
+                
+                // Ex-Dividenden-Tag 08.05.2015

```

Comment:
// // Ex-Dividenden-Tag 08.05.2015
not the same for regEx
`^([\\s]+)?EX-TAG([\\s]+){1,2}(?<exDate>\\d+.\\d+.\\d{4})([\\s]+)?$`

Reply:
changed